test/.eslintrc.json

```diff
@@ -9,5 +9,8 @@
     "expect": true,
     "waitChanged": true,
     "waitForEvent": true
+  },
+  "rules": {
+    "no-unused-expressions": "off"
```

Comment:
I don't think we should disable this rule for the entire project. Please disable the rule for the particular line, or file if there are multiple lines at stake.

Reply:
I'll remove it from global tests scope and disable the rule only for that specific test.
Later on we can revisit this topic if other tests face the same need.